Grant Application - Reference blockchain system.md

```diff
@@ -0,0 +1,286 @@
+| 0e. | Article | We will publish an article that explains the design and usage of our work in details. |
+|  1. | Specification | We will specify the configurations and parameters of the security and privacy features that protects old work against malicious new work from launching attacks on them by referring to them in our system. |
+|  2. | Setup | We provide a repository that forms the basis of our deliverables. |
+|  3. | Core Types | We provide the core data and coding types in details. |
+
+
+### Milestone 4 — Design economic model and incentive scheme
+
+- **Estimated duration:** 1 month
+- **FTE:**  1 technical researcher, and 2 economic researchers
+- **Costs:** 3,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | We will design a new reference license in our project, which records and reflects the reference relationship between the referred old works and the new works that are based on the old works. |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article that explains the design and usage of our work in details. |
+|  1. | Specification | We will specify the formalization and analysis of the economic model and incentive scheme design that collects the earnings of the new work and distributes them with all its referred works based on their reference relationships recorded on our reference system, which can motivate old works to open and share their works including internal network resources with others who can build their new work based on them. |
+
+
+### Milestone 5 — Build DAO Community
+
+- **Estimated duration:** 1 month
+- **FTE:**  1 economic researcher, and 2 management researchers
+- **Costs:** 3,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | We will design a new reference license in our project, which records and reflects the reference relationship between the referred old works and the new works that are based on the old works. |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| 0c. | Testing Guide | Core functions will be fully covered by unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
```

Comment:
DAO community is not a program, but something on Discord or Telegram like other community of Web3 projects. We will define the governance rules and operation plan of our community as deliverables in this milestone. We will also determine where it would be, whether building a website for that or using existing tools. Community is very important for our project, and can undertake several roles like filtering different versions of one work, and defending the copyright of one work.